src/tests/ChaiVagueErrorsRuleTests.ts

```diff
@@ -17,6 +17,17 @@ describe('chaiVagueErrorsRule', () : void => {
         TestHelper.assertViolations(ruleName, script, [ ]);
     });
 
+    it('should pass on xxx fluent', () : void => {
+        const script : string = `
+            expect(something, 'message').to.be.true;;
+            expect(something, 'message').to.be.false;;
+            expect(something, 'message').to.not.be.null;;
+            expect(something, 'message').to.not.be.undefined;;
```

Comment:
Style nitpick: any reason for `;;` instead of `;` here? Not a blocker at all, just sticks out a teeny bit.

Reply:
I'll remove it, but the original passing tests (`should pass on xxx`) had them, so I copied that style. Not sure why they were there in the first place.